website/client/components/groups/createPartyModal.vue

```diff
@@ -61,140 +75,120 @@ b-modal#create-party-modal(title="Empty", size='lg', hide-footer=true)
     width: 100%;
 
     .quest_screen {
-      background-image: url('~client/assets/images/quest_screen.png');
+      background-image: url('~client/assets/images/group@3x.png');
       background-size: cover;
       width: 100%;
       height: 246px;
-      margin-bottom: .5em;
+      margin-bottom: 1.5rem;
       border-radius: 2px 2px 0 0;
+      image-rendering: optimizequality;
     }
 
     h2 {
-      color: #4f2a93;
+      color: $purple-200;
     }
   }
 
-  .start-party {
-    background-image: url('~client/assets/images/basilist@3x.png');
-    background-size: cover;
-    width: 122px;
-    height: 69px;
-    margin: 0 auto;
-    margin-bottom: 1em;
+  .heading {
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+  }
+
+  input.form-control {
+    border: 0px;
+    padding-left: 0.25rem;
+  }
+
+  .input-group-prepend {
+    margin-right: 0px;
+  }
+
+  .input-group-text {
+    background-color: $white;
+    border: 0px;
+    border-radius: 0px;
+    color: $gray-300;
+    padding: 0rem 0.1rem 0rem 0.75rem;
   }
 
   .join-party {
-    background-image: url('~client/assets/images/party@3x.png');
+    background-image: url('~client/assets/images/party.png');
```

Comment:
is this going to look bad on hd displays?

Reply:
At the moment it looks bad at 1920x1080, so I'm not sure which way is preferable..

Reply:
Have you tried the @3x version?

Reply:
Yep, that's what it was before. It scales down and gets blurry.

Reply:
Hmm it looks like that with higher def images you have to specify the background-size https://stackoverflow.com/questions/16154494/retina-displays-high-res-background-images to the original size, but if it looks good, i'd say it's fine